components/profile_popover/profile_popover.jsx

```diff
@@ -188,6 +190,19 @@ class ProfilePopover extends React.PureComponent {
             this.props.hide();
         }
         this.props.actions.openModal({ModalId: ModalIdentifiers.USER_SETTINGS, dialogType: UserSettingsModal});
+        this.handleChannelMembersModalExit();
+    }
+
+    handleAddToChannel = (e) => {
+        e.preventDefault();
+
+        this.handleChannelMembersModalExit();
+    }
+
+    handleChannelMembersModalExit = () => {
```

Comment:
This method should be renamed to something a little more generic since it’s handling the closing of both the channel and team member as modals. Perhaps, `handleCloseMembersModal`.